spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -102,9 +102,11 @@
                 getSystemDefault("SPARK_YARN_JAR", "spark.yarn.jar", ""),
                 "The location of the Spark jar file. If you use yarn as a cluster, "
                 + "we should set this value")
+            .add("zeppelin.spark.useCassandraContext", "false",
+                 "Use CassandraContext instead of SQLContext if it is true")
             .add("zeppelin.spark.useHiveContext",
                 getSystemDefault("ZEPPELIN_SPARK_USEHIVECONTEXT",
-                    "zeppelin.spark.useHiveContext", "true"),
+                    "zeppelin.spark.useHiveContext", "false"),
```

Comment:
Is there any good way to keep this value unchanged? 
Changing default value might confuse user. what do you think?